interchaintest/chain_upgrade_test.go

```diff
@@ -30,9 +30,9 @@ func TestPersistenceUpgradeBasic(t *testing.T) {
 	var (
 		chainName            = "persistence"
 		upgradeRepo          = PersistenceCoreImage.Repository
-		initialVersion       = "v11.4.0"
+		initialVersion       = "v11.5.0"
```

Comment:
v11.5.2 is the latest image, not really critical there but it's better to keep the version matching currently running testnet, the next upgrade `v11.6.0` will run against it. Helps to avoid surprises.

Reply:
Resolved there by Puneet
https://github.com/persistenceOne/persistenceCore/pull/294/files